src/integration-test/integration/grouping-and-mapping/PropertiesClient.test.ts

```diff
@@ -0,0 +1,169 @@
+  });
+
+  it("Properties - Get property", async ()=> {
+    const retrievedProperty = await propertiesClient.getProperty(accessToken, mappingOne.id, groupOne.id, propertyOne.id);
+
+    expect(retrievedProperty.id).to.deep.equal(propertyOne.id);
+    expect(retrievedProperty.propertyName).to.deep.equal(propertyOne.propertyName);
+  });
+
+  it("Properties - Get all properties", async ()=> {
+    const properties = await propertiesClient.getProperties(accessToken, mappingOne.id, groupOne.id);
+    let flag = false;
+
+    for(const property of properties.properties){
+      flag = true;
+      expect([
+        propertyOne.propertyName,
+        propertyTwo.propertyName,
+        propertyThree.propertyName,
+        propertyFour.propertyName,
+      ]).to.include(property.propertyName);
+    }
+
+    expect(flag).to.be.true;
+  });
+
+  it("Properties - Get top properties", async ()=> {
+    const topProperties = await propertiesClient.getProperties(accessToken, mappingOne.id, groupOne.id, 3);
+    expect(topProperties.properties.length).to.be.equal(3);
+    expect(topProperties._links.next).not.be.undefined;
+  });
+
+  it("Properties - Get pages of properties using iterator", async ()=> {
+    const propertiesIterator = propertiesClient.getPropertiesIterator(accessToken, mappingOne.id, groupOne.id, 2);
+
+    for await(const propertiesPage of propertiesIterator.byPage()){
+      expect(propertiesPage.length).to.be.equal(2);
+      for(const property of propertiesPage){
+        expect([
+          propertyOne.propertyName,
+          propertyTwo.propertyName,
+          propertyThree.propertyName,
+          propertyFour.propertyName,
+        ]).to.include(property.propertyName);
+      }
+    }
+  });
+
+  it("Properties - Create and Delete", async ()=> {
+    const newProperty = await propertiesClient.createProperty(accessToken, mappingOne.id, groupOne.id, {
+      propertyName: "newProperty",
+      dataType: DataType.String,
+    });
+
+    expect(newProperty).not.be.undefined;
+    expect(newProperty.propertyName).to.deep.equal("newProperty");
+
+    const response = await propertiesClient.deleteProperty(accessToken, mappingOne.id, groupOne.id, newProperty.id);
+    expect(response.status).to.be.deep.equal(204);
+  });
+
+  it("Properties - Create calculated property", async ()=> {
+    const calculatedProperty = await propertiesClient.createProperty(accessToken, mappingOne.id, groupOne.id, {
+      propertyName: "BeamVolume",
+      dataType: DataType.Double,
+      quantityType: QuantityType.Volume,
+      ecProperties: [{
+        ecSchemaName: "*",
+        ecClassName: "*",
+        ecPropertyName: "Volume",
+      }],
+      calculatedPropertyType: CalculatedPropertyType.Volume,
+    });
+
+    expect(calculatedProperty).not.be.undefined;
+    expect(calculatedProperty.propertyName).to.be.equal("BeamVolume");
+    expect(calculatedProperty.calculatedPropertyType).to.be.equal(CalculatedPropertyType.Volume);
+  });
```

Comment:
Should split this test into two. `ecProperties` should be seperate from `calculatedPropertyType`

Reply:
I disagree. Do not split it. The new Grouping & Mapping API version can accept `ecProperties`, `calculatedPropertyType`, and `formula` in a single request / property.

Reply:
Sounds good. I think the name of this test can be changed.

Reply:
Changed test description to `"Properties - Create property with all parameters in a single request"`. 
Unless you meant something more like `Calculated Property - Blah..` .